packages/imperative/src/profiles/src/BasicProfileManager.ts

```diff
@@ -55,77 +51,6 @@ import { ProfileIO } from "./utils";
  * @template T
  */
 export class BasicProfileManager<T extends IProfileTypeConfiguration> extends AbstractProfileManager<T> {
-    /**
-     * Static method to initialize the profile environment. Accepts the profile root directory (normally supplied by
-     * your Imperative configuration documents) and all profile "type" configuration documents and constructs the directories
-     * needed to manage profiles of all types. You must execute this method before beginning to use profiles OR you must
-     * supply all the type configuration documents (normally obtained from your Imperative configuration document) to
-     * the constructor of
-     * @static
-     * @param {IProfileManagerInit} parms
-     * @returns {Promise<IProfileInitialized[]>}
-     * @memberof AbstractProfileManager
-     */
-    public static async initialize(parms: IProfileManagerInit): Promise<IProfileInitialized[]> {
```

Comment:
I think this will be impactful to Zowe Explorer (since they use it from the CliProfileManager which extends this class). 
It will be good to produce a ZE build with this vNext version after this PR is merged 😋 

Search for `ZoweExplorerExtender.initForZowe` in the Zowe Explorer codebase